images.css

```diff
@@ -0,0 +1,31 @@
+body {
+    text-align: center;
+}
```

Comment:
You can create a base class for this in your utility css file. This way you do not have to use text-align individually on components.